harmonica/_transformations.py

```diff
@@ -393,6 +393,69 @@ def total_gradient_amplitude(grid):
     return np.sqrt(gradient[0] ** 2 + gradient[1] ** 2 + gradient[2] ** 2)
 
 
+def tilt(grid):
+    r"""
+    Calculates the tilt of a potential field grid as defined
+    by Miller and Singh (1994)
+
+    Compute the tilt of a regular gridded potential field
+    `M`. The horizontal derivatives are calculated though finite-differences
+    while the upward derivative is calculated using FFT.
+
+    Parameters
+    ----------
+    grid : :class:`xarray.DataArray`
+        A two dimensional :class:`xarray.DataArray` whose coordinates are
+        evenly spaced (regular grid). Its dimensions should be in the following
+        order: *northing*, *easting*. Its coordinates should be defined in the
+        same units.
+
+    Returns
+    -------
+    tilt_grid : :class:`xarray.DataArray`
+        A :class:`xarray.DataArray` after calculating the
+        tilt of the passed ``grid``.
+
+    Notes
+    -----
+    The tilt is calculated as:
+
+    .. math::
+
+        tilt(f) = tan^{-1}\left(
+            \frac{
+                \frac{\partial M}{\partial z}}{
+                \sqrt{\frac{\partial M}{\partial x}^2 +
+                      \frac{\partial M}{\partial y}^2}}
+            \right)
+
+    where :math:`M` is the regularly gridded potential field.
+
+    When used on magnetic total field anomaly data, works best if the data is
+    reduced to the pole.
+
+    It's useful to plot the zero contour line of the tilt to represent possible
+    outlines of the source bodies. Use matplotlib's ``pyplot.contour`` or
+    ``pyplot.tricontour`` for this.
```

Comment:
```suggestion
    outlines of the source bodies. Use :func:`matplotlib.pyplot.contour` or
    :func:`matplotlib.pyplot.tricontour` for this.
```

This creates a link to the matplotlib docs for these 2 functions.